hooks/pre/10-reset_data.rb

```diff
@@ -56,18 +56,24 @@ def reset_candlepin
   empty_db_in_postgresql('candlepin')
 end
 
-def pg_command_base(config, command, args)
-  "PGPASSWORD='#{config[:password]}' #{command} -U #{config[:username]} -h #{config[:host]} -p #{config[:port]} #{args}"
+def pg_env(config)
+  {
+    'PGHOST' => config.fetch(:host, 'localhost'),
+    'PGPORT' => config.fetch(:port, '5432'),
+    'PGUSER' => config[:username],
+    'PGPASSWORD' => config[:password],
+    'PGDATABASE' => config[:database],
+  }
 end
 
-def pg_sql_statement(config, statement)
-  pg_command_base(config, 'psql', "-d #{config[:database]} -t -c \"#{statement}\"")
+def pg_sql_statement(statement)
+  "psql -t -c \"#{statement}\""
```

Comment:
I'd be tempted to pass in the statement via stdin but wonder if that works well with runuser (if we do that)

Reply:
runuser is only used for local sql, and there we don't need statements.

Reply:
that said, our `execute` helpers don't do stdin right now, so it'd need more refactoring.